### PR TITLE
Feature/UIScrollViewbuilder

### PR DESCRIPTION
**Pull request by mrtcelebi**

### CODE REVIEW

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        XCTAssertEqual(scrollView.backgroundColor, backgroundColor)
+    }
+    
+    func testShowsVerticalScrollIndicator() {
+        let scrollView = UIScrollViewBuilder()
+            .showsVerticalScrollIndicator(true)
+            .build()
+        
+        XCTAssertTrue(scrollView.showsVerticalScrollIndicator)
+        XCTAssertEqual(scrollView.showsVerticalScrollIndicator, true)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    }
+    
+    func testShowsHorizontalScrollIndicator() {
+        let scrollView = UIScrollViewBuilder()
+            .showsHorizontalScrollIndicator(false)
+            .build()
+        
+        XCTAssertFalse(scrollView.showsHorizontalScrollIndicator)
+        XCTAssertEqual(scrollView.showsHorizontalScrollIndicator, false)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        XCTAssertTrue(scrollView.contentInsetAdjustmentBehavior == contentInsetAdjustmentBehavior)
+    }
+    
+    func testBounces() {
+        let scrollView = UIScrollViewBuilder()
+            .bounces(true)
+            .build()
+        
+        XCTAssertTrue(scrollView.bounces)
+        XCTAssertEqual(scrollView.bounces, true)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    }
+    
+    func testAlwaysBounceVertical() {
+        let scrollView = UIScrollViewBuilder()
+            .alwaysBounceVertical(false)
+            .build()
+        
+        XCTAssertFalse(scrollView.alwaysBounceVertical)
+        XCTAssertEqual(scrollView.alwaysBounceVertical, false)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    }
+    
+    func testAlwaysBounceHorizontal() {
+        let scrollView = UIScrollViewBuilder()
+            .alwaysBounceHorizontal(true)
+            .build()
+        
+        XCTAssertTrue(scrollView.alwaysBounceHorizontal)
+        XCTAssertEqual(scrollView.alwaysBounceHorizontal, true)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    }
+    
+    func testIsPagingEnabled() {
+        let scrollView = UIScrollViewBuilder()
+            .isPagingEnabled(false)
+            .build()
+        
+        XCTAssertFalse(scrollView.isPagingEnabled)
+        XCTAssertEqual(scrollView.isPagingEnabled, false)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    }
+    
+    func testIsScrollEnabled() {
+        let scrollView = UIScrollViewBuilder()
+            .isScrollEnabled(true)
+            .build()
+        
+        XCTAssertTrue(scrollView.isScrollEnabled)
+        XCTAssertEqual(scrollView.isScrollEnabled, true)
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: We can remove that.
```suggestion
```

File: Example/Tests/UIScrollViewBuilderTests.swift [hunk 8]
```diff
@@ -0,0 +1,124 @@
+//
+//  UIScrollViewBuilderTests.swift
+//  MobilliumBuilders_Tests
+//
+//  Created by Murat Celebi on 17.03.2021.
+//  Copyright © 2021 CocoaPods. All rights reserved.
```
Comment (alihanaktay) [inline Example/Tests/UIScrollViewBuilderTests.swift]: Copyright should be Mobillium.
```suggestion
//  Copyright © 2021 Mobillium. All rights reserved.
```

File: MobilliumBuilders/Classes/UIScrollViewBuilder.swift [hunk 9]
```diff
@@ -0,0 +1,93 @@
+//
+//  UIScrollViewBuilder.swift
+//  MobilliumBuilders
+//
+//  Created by Murat Çelebi on 17.03.2021.
+//
```
Comment (alihanaktay) [inline MobilliumBuilders/Classes/UIScrollViewBuilder.swift:7]: We can add copyright
```suggestion
// Copyright © 2021 Mobillium. All rights reserved.
```

File: MobilliumBuilders/Classes/UIStackViewBuilder.swift [hunk 10]
```diff
@@ -1,5 +1,5 @@
 //
-//  NSAttributedStringBuilder.swift
+//  UIStackViewBuilder.swift
 //  MobilliumBuilders
 //
 //  Created by Murat Çelebi on 17.03.2021.
```
Comment (alihanaktay) [inline MobilliumBuilders/Classes/UIStackViewBuilder.swift]: We can add copyright

**Review by alihanaktay — CHANGES_REQUESTED** [review]

**Review by alihanaktay — APPROVED** [review]

**Review by aslanmehmetsalih — APPROVED** [review]

**Review by aslanmehmetsalih — APPROVED** [review]

**Pull request merged**